.gitignore

```diff
@@ -405,7 +405,8 @@ FodyWeavers.xsd
 tmp/
 temp/
 .fsdocs
-docs/
+docs/*
+!docs/Makefile
```

Comment:
@webwarrior-ws what is this change for?

Reply:
It works now. Have to run `make` with `--directory` argument.

Reply:
So abandon 2nd commit and use initial approach?